<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvUpSummaryWidgetSmall</class>
 <widget class="QWidget" name="C_SyvUpSummaryWidgetSmall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>C_SyvUpSummaryWidgetSmall</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>18</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeLabSvgOnly" name="pc_LabelTitleIcon">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabelTitle">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeFrameSeparator" name="frame">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabelPercent">
     <property name="text">
      <string>P%</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeWiProgressBar" name="pc_ProgressBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>8</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>8</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabelTime">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeFrameSeparator" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabelEstimated">
     <property name="text">
      <string>Estimated</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeFrameSeparator" name="pc_SeparatorRight">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabelDataTransfer">
     <property name="text">
      <string>DataTransfer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgePubSvgIconOnly" name="pc_PushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeFrameSeparator</class>
   <extends>QFrame</extends>
   <header>C_OgeFrameSeparator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeWiProgressBar</class>
   <extends>QWidget</extends>
   <header>C_OgeWiProgressBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGenericNoPaddingNoMargins.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubSvgIconOnly</class>
   <extends>QPushButton</extends>
   <header>C_OgePubSvgIconOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabSvgOnly</class>
   <extends>QLabel</extends>
   <header>C_OgeLabSvgOnly.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
